### DISCUSSION TITLE
BlueRetro Core module: disabel LEDs while propper use.

### DISCUSSION THREAD

**Discussion by wrenchdog** [darthcloud/BlueRetro#1016] [Ideas]
Hi folks.
I am using two devices of the BlueRetro Core. Is it somehow possible to disable both green and blue LED? Only the blue should do pulsing light when connecting with controllers. 

LED on when error is okay too but in normal use both LED, basicly the green one is on, are so super bright. It is a bit disruptive when playing games.

Do you see any possibility to work on this on firmware side?

Best Regards.

**Comment by darthcloud** [comment]
I'm not the one making those device, but I'm pretty sur they are hard wire. Your best bet is to put tape on it to hide them

**Comment by wrenchdog** [comment]
Thanks for reply. Solved it with dark acrylic glas casing.
